toolkit/tools/imagegen/installutils/installutils.go

```diff
@@ -37,6 +37,9 @@ const (
 	// rpmDependenciesDirectory is the directory which contains RPM database. It is not required for images that do not contain RPM.
 	rpmDependenciesDirectory = "/var/lib/rpm"
 
+	// rpmManifestDirectory is the directory containing manifests of installed packages
```

Comment:
Please extend comment "...to support distroless scanning tools"

Reply:
Updated